ecommerce/products/views.py

```diff
@@ -0,0 +1,5 @@
+from django.http import HttpResponse
+
+
+def index(request):
+    return HttpResponse("Hello, world. You're at the products index.")
```

Comment:
Nice one, this works as expected

![Screenshot 2021-02-24 at 15 17 20](https://user-images.githubusercontent.com/11095240/109022129-70492f00-76b3-11eb-970c-50b216325702.png)

Reply:
B)